nds.go

```diff
@@ -54,6 +54,7 @@ func init() {
 	gob.Register(&datastore.Key{})
 	gob.Register(datastore.GeoPoint{})
 	gob.Register(&datastore.Entity{})
+	gob.Register([]interface{}{})
```

Comment:
Here's the bugfix - after adding the expected log checks to the test, the `TestGetSliceProperty` test kept failing with the following log:

`nds_test.go:164: gob: type not registered for interface: []interface {}`

It its my belief this was the proper fix for it.